services/scraper/scraper_test.go

```diff
@@ -0,0 +1,50 @@
+package scraper_test
+
+import (
+	"fmt"
+	"net/url"
+
+	"go-google-scraper-challenge/initializers"
+	"go-google-scraper-challenge/models"
+	"go-google-scraper-challenge/services/scraper"
+	. "go-google-scraper-challenge/tests"
+	. "go-google-scraper-challenge/tests/helpers"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Scraper", func() {
+	Describe("#Search", func() {
+		Context("given valid keywords", func() {
+			BeforeEach(func() {
+				keyword := "keyword"
+				visitURL := fmt.Sprintf("http://www.google.com/search?q=%s", url.QueryEscape(keyword))
+				cassetteName := "scraper/success"
+
+				RecordResponse(cassetteName, visitURL)
+			})
+
+			It("creates a result with the given keyword", func() {
+				user := FabricateUser("dev@nimblehq.co", "password")
+				keywords := []string{"keyword"}
+				scraper.Search(keywords, user)
+
+				userResults, err := models.GetResultsByUserId(user.Id)
+				if err != nil {
+					Fail("Failed to get user results")
+				}
+
+				Expect(userResults).To(HaveLen(1))
+				Expect(userResults[0].Keyword).To(Equal("keyword"))
+			})
+		})
+	})
+
+	AfterEach(func() {
+		initializers.CleanupDatabase("users")
```

Comment:
Is it possible to adjust  `CleanupDatabase ` function so that it takes array/slice of tables? then we have to call it once ex: `initializers.CleanupDatabase(clean-up-tables-array/slice)`

Reply:
@junan ✅  updated in [4473091](https://github.com/carryall/go-google-scraper-challenge/pull/40/commits/4473091605a3ccf3eb15b9be90dead792011e018)